recipes/cellpy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "cellpy" %}
+{% set version = "0.4.0a4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 722ebe3dc25e82b74b9a7538418d3ac32a31658d1544e4110b7058315c7e82de
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - cellpy=cellpy.cli:cli
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip >=20.0.2
+    - setuptools >=38.6.0
+  run:
+    - python >=3.6
+    - numpy >=1.15
+    - xlrd >=1.2.0
+    - matplotlib >=3.1.3
```

Comment:
Sorry, just noticed this. Depending on `matplotlib` in conda-forge currently means "install the matplotlib library as well as PyQt". Do you really need a GUI? Otherwise, you can change this to `matplotlib-base`.

Reply:
Hi. No, GUI is not needed. I will change to matplotlib-base. Thanks.